addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+    spec:
+      tag: <DOCKER-IMAGE-TAG>
+      serviceAccountName: tutorial-service-account
+      builder:
+        name: my-builder
+        kind: Builder
+      source:
+        git:
+          url: https://github.com/spring-projects/spring-petclinic
+          revision: 82cb521d636b282340378d80a6307a08e3d4a4c4
+    ```
+
+    - Replace `<DOCKER-IMAGE-TAG>` with a valid image tag that exists in the registry you configured with the `--docker-server` flag when creating a Secret in step #1. Something like: `your-name/app` or `gcr.io/your-project/app`
+    - If you are using your application source, replace `source.git.url` & `source.git.revision`.
+   > Note: To use a private git repo follow the instructions in [secrets](secrets.md)
+
+   Apply that image resource to the cluster
+
+    ```bash
+    kubectl apply -f image.yaml
+    ```
+
+   You can now check the status of the image resource.
+
+   ```bash
+   kubectl -n default get images
+   ```
+
+   You should see that the image resource has an unknown READY status as it is currently building.
+
+   ```
+    NAME                  LATESTIMAGE   READY
+    tutorial-image                      Unknown
+    ```
+
+   You can tail the logs the image that is currently building using the [kp cli](https://github.com/vmware-tanzu/kpack-cli/blob/main/docs/kp_build_logs.md)
+
+    ```
+    kp build logs tutorial-image -n default
+    ``` 
+
+   Once the image resource finishes building you can get the fully resolved built OCI image with `kubectl get`
+
+    ```
+    kubectl -n default get image tutorial-image
+    ```  
+
+   The output should look something like this:
+    ```
+    NAMESPACE   NAME                  LATESTIMAGE                                        READY
+    default     tutorial-image        index.docker.io/your-project/app@sha256:6744b...   True
+    ```
+
+   The latest OCI image is available to be used locally via `docker pull` and in a kubernetes deployment.
+
+8. Run the built app locally
+
+   Download the latest OCI image available in step #6 and run it with docker.
+
+   ```bash
+   docker run -p 8080:8080 <latest-image-with-digest>
+   ```
+
+   You should see the java app start up:
+   ```
+       
+              |\      _,,,--,,_
+             /,`.-'`'   ._  \-;;,_
+    _______ __|,4-  ) )_   .;.(__`'-'__     ___ __    _ ___ _______
+    |       | '---''(_/._)-'(_\_)   |   |   |   |  |  | |   |       |
+    |    _  |    ___|_     _|       |   |   |   |   |_| |   |       | __ _ _
+    |   |_| |   |___  |   | |       |   |   |   |       |   |       | \ \ \ \
+    |    ___|    ___| |   | |      _|   |___|   |  _    |   |      _|  \ \ \ \
+    |   |   |   |___  |   | |     |_|       |   | | |   |   |     |_    ) ) ) )
+    |___|   |_______| |___| |_______|_______|___|_|  |__|___|_______|  / / / /
+    ==================================================================/_/_/_/
+    
+    :: Built with Spring Boot :: 2.2.2.RELEASE
+   ``` 
+
+9. kpack rebuilds
```

Comment:
For consistency as the other steps open with a sentence, you could change this to 'Updating the kpack image resource' or 'Rebuilding the kpack image'